src/js/views/admin/sidebar/clinician/clinician-sidebar_views.js

```diff
@@ -91,79 +140,160 @@ const SidebarView = View.extend({
+    const isDisabled = this.clinician.isNew();
+    const accessComponent = new AccessComponent({ model: this.clinician, state: { isDisabled } });
 
     this.listenTo(accessComponent, 'change:access', accessType => {
-      this.model.save({ access: accessType });
+      this.clinician.save({ access: accessType });
     });
 
     this.showChildView('access', accessComponent);
   },
   showRole() {
-    const isDisabled = this.model.isNew();
-    const roleComponent = new RoleComponent({ model: this.model, state: { isDisabled } });
+    const isDisabled = this.clinician.isNew();
+    const roleComponent = new RoleComponent({ model: this.clinician, state: { isDisabled } });
 
     this.listenTo(roleComponent, 'change:role', role => {
-      this.model.saveRole(role);
+      this.clinician.saveRole(role);
     });
 
     this.showChildView('role', roleComponent);
   },
   showGroups() {
-    this.showChildView('groups', new GroupsComponent({
-      clinician: this.model,
-    }));
+    this.showChildView('groups', new GroupsComponent({ clinician: this.clinician }));
   },
   showSave() {
-    if (!this.model.isValid()) {
+    if (!this.clonedClinician.isValid() || !_.isEmpty(this.state.get('errors'))) {
       this.showDisabledSave();
       return;
     }
 
-    this.showChildView('save', new SaveView({ model: this.model }));
+    this.showChildView('save', new SaveView({ model: this.clonedClinician }));
   },
   showDisabledSave() {
     this.showChildView('save', new DisabledSaveView());
   },
   showForm() {
-    if (this.model.isNew()) {
-      this.listenTo(this.model, 'change:name change:email', this.showSave);
+    if (_.isEmpty(this.state.get('errors'))) {
+      this.cloneClinician();
+    }
+
+    if (this.clinician.isEditable()) {
+      this.listenTo(this.clonedClinician, {
+        'change:name'() {
+          if (this.state.get('errors')) {
+            this.state.set({ errors: _.omit(this.state.get('errors'), 'name') });
+          }
+          this.showSave();
+        },
+        'change:email'() {
+          if (this.state.get('errors')) {
+            this.state.set({ errors: _.omit(this.state.get('errors'), 'email') });
+          }
+          this.showSave();
+        },
+      });
       this.showDisabledSave();
     }
 
     this.showName();
     this.showEmail();
   },
   showInfo() {
-    if (this.model.isNew()) return;
+    if (this.clinician.isNew()) return;
 
-    if (!this.model.get('_role') || this.model.getGroups().length === 0) {
+    if (!this.clinician.get('_role') || this.clinician.getGroups().length === 0) {
       this.showChildView('info', new InfoView());
       return;
     }
 
     this.getRegion('info').empty();
   },
+  showErrors(errors) {
+    this.state.set({ errors });
+    this.showName(errors.name);
+    this.showEmail(errors.email);
+    this.showSave();
+  },
   onSave() {
-    this.getRegion('save').empty();
+    this.state.set({ errors: null });
```

Comment:
would the save still empty?

Reply:
Move this to `showSave` to avoid having the button disappear and reappear when errors happen.